### DISCUSSION TITLE
What is the browser Support

### DISCUSSION THREAD

**Discussion by ajaybeniwal** [pmndrs/valtio#77] [General]
Is the IE11 supported by Valtio ?

**Comment by dai-shi** [comment]
Unfortunately not. Valtio depends heavily on Proxy API. Even with proxy-polyfill, IE11 is hardly supported.

**Reply by Noitidart** [reply]
Aw drats. Which polyfill's will give the best legacy browser support we can get?

**Reply by dai-shi** [reply]
Probably most proxy polyfills depend on object getters and setters, so adding new properties won't be detected. You can try any of them to see how it behaves.
Also, WeakMap polyfill will be required, which might be more tough.
